charts/everest/charts/common/templates/_assign_default_psp.yaml.tpl

```diff
@@ -0,0 +1,150 @@
+#
+# @param .namespace     The namespace where Everest server is installed
+#
+{{- define "everest.assignDefaultPsp" }}
+{{- $hookName := printf "everest-helm-psp-post-upgrade-hook" }}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ $hookName }}
+  namespace: {{ .namespace }}
+  annotations:
+    "helm.sh/hook": post-upgrade
```

Comment:
Did you test that this actually works (does not result in restarts?)

This hook is run _after_ the upgrade is complete, meaning that everest-operator would have upgraded to 1.7.0 before this hooks run. Wouldn't it have had the time to run at least one reconciliation where it changes the affinity to `nil`, before this hook kicks in?

Reply:
updated, now default PSPs are assigned without restarting DB clusters